src/include/miopen/cat/problem_description.hpp

```diff
@@ -84,24 +84,22 @@ struct ProblemDescription : ProblemDescriptionBase
 
             if(ydims.size() != xdims.size())
             {
-                MIOPEN_THROW(miopenStatusBadParm,
-                             "CatForward: Tensor dimension lengths do not match.");
+                return false;
```

Comment:
Ditto

Reply:
changed it.

Reply:
See https://github.com/ROCm/MIOpen/pull/2562#discussion_r1437751964

Reply:
move it to constructor, too